plugins/woocommerce/client/blocks/assets/js/data/cart/thunks.ts

```diff
@@ -495,11 +495,20 @@ export const updateCustomerData =
 		// Address data to be updated; can contain both billing_address and shipping_address.
 		customerData: Partial< BillingAddressShippingAddress >,
 		// If the address is being edited, we don't update the customer data in the store from the response.
-		editing = true
+		editing = true,
+		haveAddressFieldsForShippingRatesChanged = false
 	) =>
 	async ( { dispatch }: CartThunkArgs ) => {
 		try {
 			dispatch.updatingCustomerData( true );
```

Comment:
We are still calling `updatingCustomerData` with each field change; we're only flagging when shipping fields that impact shipping rates have changed without changing the server update logic.